src/indexer/utils/substrate/index.ts

```diff
@@ -137,6 +139,9 @@ export async function saveDeposit(
       timestamp: timestamp,
       destination: `0x${depositData.substring(2).slice(128, depositData.length - 1)}`,
     } as Pick<DecodedDepositLog, "depositNonce" | "sender" | "amount" | "destination" | "resourceID" | "toDomainId" | "fromDomainId" | "timestamp">
+
```

Comment:
Is this missing an adress status?

Reply:
isn't OFAC just for EVM addresses?

Reply:
If we decided for this field to represent address status (we are not using boolean), we should also then add it on the substrate side

Reply:
I have added this